modules/programs/fish.nix

```diff
@@ -349,6 +349,18 @@ in {
               config.home.sessionVariables)
           }
 
+          for p in "$HOME/.nix-profile/bin"
```

Comment:
Why a for loop over one item? If you want to bind it to prevent repetition, you could just use `set -l` and parens:

```
(
  set -l p "$HOME/.nix-profile/bin"
  if not contains $p $PATH
    set -p PATH $p
  end
)
```

Reply:
I was thinking for extensibily it would be easy to add to this.
Even though I couldn't envision an extension.

Reply:
adding the brackets I was running in a command substitution not allowed error, so I removed them

Reply:
Indeed, I forgot about that x)